src/main/java/chat/commands/roleDistribution/RegisterRole.java

```diff
@@ -0,0 +1,95 @@
+package chat.commands.roleDistribution;
```

Comment:
no capital characters in packages, if you really want a "space" there, use a _.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html